### PR TITLE
Ensure audience estimates don't return values over 100%

### PR DESCRIPTION
**Pull request by roborourke**
Fixes https://github.com/humanmade/altis-analytics/issues/88

### CODE REVIEW

File: inc/audiences/namespace.php [hunk 1]
```diff
@@ -547,13 +558,14 @@ function get_unique_endpoint_count() : ?int {
 	];
 
 	$cache = wp_cache_get( 'total-uniques', 'altis-audiences' );
-	if ( $cache ) {
+	if ( $cache && ! $force_update ) {
```
Comment (jazzsequence) [inline inc/audiences/namespace.php:561]: If the issue has to do with cached values, and we're passing `$force_update` to bypass the cache, should we not cache the new, updated values? Is the cache set elsewhere?

Comment (roborourke) [reply inc/audiences/namespace.php:561]: Yeah further down in the code. Before this change if the `$cache` value was truthy we'd just return it early here. If the function continues on the query is re-run and then the cache is set at the end of the function before returning the value.

Comment (jazzsequence) [reply inc/audiences/namespace.php:561]: I figured, since I didn't see the cache set in this code, I just wanted to make sure that was the case.

**Review by jazzsequence — APPROVED** [review]
👍

File: inc/audiences/namespace.php (hunk 1 above)
Comment (roborourke) [reply inc/audiences/namespace.php:561]: FWIW I regularly use the little blue up/down arrows in the code review view to look at an entire function, or sometimes a while file to give myself context.

Comment (jazzsequence) [reply inc/audiences/namespace.php:561]: Yeah, I did that, but I still didn't see where the cache was being set in this path. Am I missing it?

Comment (roborourke) [reply inc/audiences/namespace.php:561]: https://github.com/humanmade/aws-analytics/pull/146/files#diff-2667fc93072cbbc8eb88479fb4b4e173e128dc672084113b9c5c297e7c92db67R574

Comment (jazzsequence) [reply inc/audiences/namespace.php:561]: I missed it. See it now. 👍

**Pull request merged**